### PR TITLE
minor: fixed TeamCity violation

### PR DESCRIPTION
**Pull request by shashwatj07**
https://teamcity.jetbrains.com/viewLog.html?buildId=3524344&tab=Inspection&buildTypeId=Checkstyle_IdeaInspectionsMaster

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java [hunk 1]
```diff
@@ -66,7 +65,7 @@ private static String getPackageFromFilePath(String filePath) {
     private static List<String> readFile(Path filePath) throws CheckstyleException {
         try (BufferedReader br = Files.newBufferedReader(
                 filePath, StandardCharsets.UTF_8)) {
-            return new ArrayList<>(br.lines().collect(Collectors.toList()));
+            return br.lines().collect(Collectors.toList());
```
Comment (strkkk) [inline src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java:68]: `return Files.readAllLines(filePath)` would be even simpler

Comment (shashwatj07) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java]: done

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java [hunk 2]
```diff
@@ -68,14 +64,8 @@ private static String getPackageFromFilePath(String filePath) {
...
-    private static List<String> readFile(Path filePath) throws CheckstyleException {
-        try (BufferedReader br = Files.newBufferedReader(
-                filePath, StandardCharsets.UTF_8)) {
-            return new ArrayList<>(br.lines().collect(Collectors.toList()));
-        }
-        catch (IOException ex) {
-            throw new CheckstyleException("Failed to read " + filePath, ex);
-        }
+    private static List<String> readFile(Path filePath) throws IOException {
+        return Files.readAllLines(filePath);
```
Comment (strkkk) [inline src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java:68]: you have changed method signature, which is not a good idea in general.
Before it was throwing CheckstyleException insted of IOException, please leave it as before and only fix the convertion to list part.

**Review by strkkk — CHANGES_REQUESTED** [review]

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java (hunk 2 above)
Comment (shashwatj07) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java]: Please don't apologize. I've pushed changes.

**Review by strkkk — APPROVED** [review]

**Pull request merged**